src/Qir/qir-utils.ps1

```diff
@@ -34,163 +34,41 @@ function Build-CMakeProject {
-
-            # Address Sanitizer (https://clang.llvm.org/docs/AddressSanitizer.html)
-            # Win: (Conflict between the ASan library and MSVC library)
-            #   [19/35] Linking CXX shared library lib\QIR\Microsoft.Quantum.Qir.Runtime.dll
-            #   FAILED: lib/QIR/Microsoft.Quantum.Qir.Runtime.dll lib/QIR/Microsoft.Quantum.Qir.Runtime.lib
-            #   cmd.exe /C "cd . && C:\PROGRA~1\LLVM12\bin\CLANG_~1.EXE -fuse-ld=lld-link -nostartfiles -nostdlib -Werror -Weverything .... \
-            #       -fsanitize=address -g -Xclang -gcodeview -O0 -DDEBUG -D_DEBUG -D_DLL -D_MT -Xclang --dependent-lib=msvcrtd  \
-            #       -Xlinker /guard:cf -shared -o lib\QIR\Microsoft.Quantum.Qir.Runtime.dll  -Xlinker /implib:lib\QIR\Microsoft.Quantum.Qir.Runtime.lib \
-            #       -Xlinker /pdb:lib\QIR\Microsoft.Quantum.Qir.Runtime.pdb -Xlinker /version:0.0 lib/QIR/bridge-rt.obj \
-            #       lib/QIR/CMakeFiles/qir-rt-support-obj.dir/QirRange.cpp.obj lib/QIR/CMakeFiles/qir-rt-support-obj.dir/OutputStream.cpp.obj ....\
-            #       -lkernel32 -luser32 -lgdi32 -lwinspool -lshell32 -lole32 -loleaut32 -luuid -lcomdlg32 -ladvapi32 -loldnames && cd ."
-            #   lld-link: error: duplicate symbol: malloc
-            #   >>> defined at C:\src\llvm_package_6923b0a7\llvm-project\compiler-rt\lib\asan\asan_win_dll_thunk.cpp:34
-            #   >>>            clang_rt.asan_dll_thunk-x86_64.lib(asan_win_dll_thunk.cpp.obj)
-            #   >>> defined at ucrtbased.dll
-            #   clang++: error: linker command failed with exit code 1 (use -v to see invocation)
-            $sanitizeFlags += " -fsanitize=address"   # https://clang.llvm.org/docs/AddressSanitizer.html
-            #   TODO:
-            #       * Some tests verify the failure behavior, i.e. they cause `Fail()` to be called and return to the caller with the exception. 
-            #         Any allocations made between the call and the exception throw (caught by `REQUIRE_THROWS()`) are leaking.
-            #         Extract such tests to a separate .cpp file or executable and compile with leak check off (or suppress leaks in that .cpp or executable only).
-
+        if (-not ($IsWindows)) {
             # Common for all sanitizers:
-            $sanitizeFlags += " -fsanitize-blacklist="      # https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html#suppressing-errors-in-recompiled-code-ignorelist
-                                                            # https://releases.llvm.org/11.0.1/tools/clang/docs/SanitizerSpecialCaseList.html
+            $sanitizeFlags = " -fsanitize-blacklist="      # https://clang.llvm.org/docs/UndefinedBehaviorSanitizer.html#suppressing-errors-in-recompiled-code-ignorelist
+            # https://releases.llvm.org/11.0.1/tools/clang/docs/SanitizerSpecialCaseList.html
             $sanitizeFlags += (Join-Path $Path .. UBSan.ignore)
```

Comment:
Is not having this flag in the CMakeLists.txt going to interfere with the sanitizer behavior when compiling directly through there? Would it be worth putting this path into a define variable that is then used in the CMakeLists.txt?